connectors/sources/sharepoint_server.py

```diff
@@ -35,13 +35,13 @@
 SELECTED_FIELDS = "WikiField, Modified,Id,GUID,File,Folder"
 
 URLS = {
-    PING: "{host_url}{parent_site_url}_api/web/webs",
-    SITES: "{host_url}{parent_site_url}/_api/web/webs?$skip={skip}&$top={top}",
-    LISTS: "{host_url}{parent_site_url}/_api/web/lists?$skip={skip}&$top={top}&$expand=RootFolder&$filter=(Hidden eq false)",
```

Comment:
What's the difference between Hidden and NoCrawl? Do we want both to be eq false?

Reply:
This is the field that SharePoint Search uses to decide, so I thought it would be a better approach. The results are very similar.

Reply:
Since the connector is also used by other customers, it should not break their flows - if it's not identical, we need to be very very careful.

I'd leave it as is if it works for you, if not we can see and make this a configurable option via configurable fields.

Reply:
Changed back to Hidden, but will add this option in a different pull request.